assets/src/components/cd/cluster/pod/PodInfo.tsx

```diff
@@ -25,7 +38,34 @@ function ViewLogsButton() {
   )
 }
 
+export const ColActions = ({
+  clusterId,
+  podName,
+  namespace,
+}: {
+  clusterId?: string
+  podName?: string
+  namespace?: string
+}) =>
```

Comment:
I've discovered that creating table columns as functions this way can cause weird effects like Modals closing when the underlying data changes even when memoizing the columns, so I've started passing extra data in through the `meta` prop of react-table. Here's a basic example:

```
export const ColActions = columnHelper.accessor((row) => row.name, {
  id: 'actions',
  cell: function Cell({ table, row }) {
    const { refetch, outsideId } = table.options.meta as any

    return (
      <Button onClick={() => refetch?.()}>
        {outsideId}: {row.original.name}
      </Button>
    )
  },
})

const data = [
  { id: '1', name: 'name1' },
  { id: '1', name: 'name1' },
]

const columns = [ColActions]

export function ATable() {
  return (
    <Table
      data={data}
      columns={columns}
      reactTableOptions={{
        meta: {
          outsideId: 'outside-id',
          refetch: () => {
            console.log('refetch')
          },
        },
      }}
    />
  )
}
```

Reply:
Good to know, I will update it tomorrow.

Reply:
Good to know, thanks. I will update it tomorrow.